frontend/packages/shared/src/components/AltinnSpinner.tsx

```diff
@@ -12,7 +12,7 @@ export type IAltinnSpinnerComponentProvidedProps = {
 export const AltinnSpinner = ({
   spinnerText,
   className,
-  size = '1xLarge',
+  size = 'xlarge',
```

Comment:
I suggest using `medium` as `xlarge` seems too big in some cases. Here is an example on the create service page:

**Before (1xLarge):**
<img width="649" alt="old-spinner" src="https://github.com/Altinn/altinn-studio/assets/24462611/4039ebfb-3640-4909-bae1-27522354981b">

**After (xlarge):**
<img width="649" alt="new-spinner" src="https://github.com/Altinn/altinn-studio/assets/24462611/5a34c7f8-5d15-4237-8972-10d261c5b07a">

```suggestion
  size = 'medium',
```